src/loggerBuilder.ts

```diff
@@ -0,0 +1,52 @@
+import { createLogger, format } from 'winston';
+import Transport from 'winston-transport';
+
+const { combine, splat, timestamp, printf } = format;
+
+const myFormat = printf(info => {
+    let msg = `${info.message} ${info.timestamp} `;
+    if (info.meta) {
+        msg += JSON.stringify(info.meta);
+    }
+    return msg;
+});
+
+class SimpleConsole extends Transport {
+    log(info: any, callback: () => void) {
+        setImmediate(() => this.emit('logged', info));
+        const msg = info[Symbol.for('message')];
+
+        // Use console here so request ID and log level can be automatically attached in CloudWatch log
+        switch (info[Symbol.for('level')]) {
+            case 'debug':
+                console.debug(msg);
+                break;
+            case 'info':
+                console.info(msg);
+                break;
+            case 'warn':
+                console.warn(msg);
+                break;
+            case 'error':
+                console.error(msg);
+                break;
+            default:
+                console.log(msg);
+                break;
+        }
+
+        if (callback) {
+            callback();
+        }
+    }
+}
+
+// eslint-disable-next-line import/prefer-default-export
+export function makeLogger(metadata?: any) {
+    return createLogger({
+        level: process.env.LOG_LEVEL,
+        format: combine(format.colorize(), splat(), timestamp(), format.json(), myFormat),
```

Comment:
It's probably best to remove `format.colorize()`. Colors look pretty in the local console but I think that they'll end up as useless escape codes in cloudwatch. Make sure to check the resulting error log format in cloudwatch.

Reply:
Got it! Yea, I've been fighting with formatting in CloudWatch yesterday all day >_<, it's completely different from what's showed up locally. I'll remove the colorize part.